operator/v1/types_ingress.go

```diff
@@ -99,6 +99,13 @@ type IngressControllerSpec struct {
 	// the generated certificate's CA will be automatically integrated with the
 	// cluster's trust store.
 	//
+	// If a wildcard certificate is used and shared by multiple
+	// HTTP/2 enabled routes (which implies ALPN) then clients
+	// (i.e., notably browsers) are at liberty to reuse open
+	// connections. This means a client can reuse a connection to
+	// another route and that is likely to fail. This behaviour is
+	// generally known as connection coalescing.
```

Comment:
Should we add a statement along the lines of the following?  "Therefore HTTP/2 not enabled for routes that use this default certificate; that is, a route must specify its own certificate to enable HTTP/2."  I see you're adding the information to the route godoc, but it bears repeating here for completeness of the `defaultCertificate` godoc.

Reply:
There are also implications for router sharding, but maybe those don't need to be stated explicitly (an those implications are likely to be exactly the desirable consequences that connection coalescing aims to achieve).

Reply:
> Therefore HTTP/2 not enabled for routes that use this default certificate; that is, a route must specify its own certificate to enable HTTP/2.

All good, but what's "this" referring to?

Reply:
The certificate referenced by `defaultCertificate`.